xla/service/gpu/nccl_clique_key.cc

```diff
@@ -37,8 +37,10 @@ namespace xla::gpu {
 //===----------------------------------------------------------------------===//
 
 NcclCliqueKey::NcclCliqueKey(std::vector<GlobalDeviceId> devices,
-                             int64_t stream_id)
-    : devices_(std::move(devices)), stream_id_(stream_id) {}
+                             int64_t stream_id, AsyncStreamKind stream_kind)
```

Comment:
this looks like a part of different change

Reply:
we need this when creating the clique key so we know what config to use when using the key to create comm.